loafwallet/BuyTableViewController.swift

```diff
@@ -28,7 +28,7 @@ class BuyTableViewController: UITableViewController, SFSafariViewControllerDeleg
 	@IBOutlet var moonpaySegmentedControl: UISegmentedControl!
 
 	@IBAction func didTapMoonpay(_: Any) {
-		let timestamp = Int(appInstallDate.timeIntervalSince1970)
+		let timestamp = Int(Date().timeIntervalSince1970)
```

Comment:
TBH, I think this was a code snippet from Moonpay to speed integration and my guess is they are using it as a verification @Iferencak